NodeTest/NodeTest.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+require('dotenv').config();
+
+//create new express app and save it in 'app'
+const app = express();
+const port = process.env.PORT || 8000;
+
+
+//Call the route here from the external file
+let routes = require ('./Route')(app);
```

Comment:
I think you should do this:
```suggestion
const routes = require ('./Route');

routes(app);
```